src/software/ai/hl/stp/tactic/chip/simulated_chip_tactic_test.cpp

```diff
@@ -53,27 +53,28 @@ TEST_P(SimulatedChipTacticTest, chip_test)
             non_terminating_validation_functions, Duration::fromSeconds(5));
 }
 
-INSTANTIATE_TEST_CASE_P(BallLocations, SimulatedChipTacticTest,
-                        ::testing::Values(
-                            // place the ball directly to the left of the robot
-                            std::make_tuple(Vector(0, 0.5), Angle::zero()),
-                            // place the ball directly to the right of the robot
-                            std::make_tuple(Vector(0, -0.5), Angle::zero()),
-                            // place the ball directly infront of the robot
-                            std::make_tuple(Vector(0.5, 0), Angle::zero()),
-                            // place the ball directly behind the robot
-                            std::make_tuple(Vector(-0.5, 0), Angle::zero()),
-                            // place the ball in the robots dribbler
-                            std::make_tuple(Vector(ROBOT_RADIUS, 0), Angle::zero()),
+INSTANTIATE_TEST_CASE_P(
+    BallLocations, SimulatedChipTacticTest,
+    ::testing::Values(
+        // place the ball directly to the left of the robot
+        std::make_tuple(Vector(0, 0.5), Angle::zero()),
+        // place the ball directly to the right of the robot
+        std::make_tuple(Vector(0, -0.5), Angle::zero()),
+        // place the ball directly infront of the robot
+        std::make_tuple(Vector(0.5, 0), Angle::zero()),
+        // place the ball directly behind the robot
+        std::make_tuple(Vector(-0.5, 0), Angle::zero()),
+        // place the ball in the robots dribbler
+        std::make_tuple(Vector(ROBOT_MAX_RADIUS_METERS, 0), Angle::zero()),
```

Comment:
`ROBOT_MAX_RADIUS_METERS` isn't injected here? not a criticism, just trying to understand why not

Reply:
this happens in a few other places too

Reply:
it doesn't seem worth it since it's hard to do especially with obstacle generation

Reply:
Interesting, what's the gain we get from dependency injecting in other places in that case? Is it just not worth it here? 

(this is definitely not blocking)

Reply:
dependency injecting into sw is good for trajectory planning in sw + potential hybrid fleet, and these values seem like they are likely to change from one robot revision to the next.

Reply:
Ah, so we're just using the static constants for sim for now?

Reply:
what was the reason for having different constants?

Reply:
For sim? I might be missing context here but DI is useful in sim mostly for testing. For most physics / sim implementation code we don't want tests to fail if we tweak values to make them more realistic for gameplay.